manim/mobject/svg/brace.py

```diff
@@ -3,6 +3,7 @@
 __all__ = ["Brace", "BraceLabel", "BraceText", "BraceBetweenPoints"]
 
 
+from manim.utils.color import BLACK
```

Comment:
Make this a relative import, and club together with other library imports (i.e. L9-L17)

Reply:
Can you explain this to me? I don't know what you mean, sorry.

Reply:
Ok, got it :D